mmv1/products/networksecurity/SecurityProfileGroup.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'SecurityProfileGroup'
+base_url: '{{parent}}/locations/{{location}}securityProfileGroups'
+create_url: '{{parent}}/locations/{{location}}/securityProfileGroups?securityProfileGroupId={{name}}'
+self_link: '{{parent}}/locations/{{location}}/securityProfileGroups/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  A security profile group defines a container for security profiles.
+references:
+  !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/firewall/docs/reference/network-security/rest/v1beta1/organizations.locations.securityProfileGroups'
+  guides:
+    'Security profile groups overview': 'https://cloud.google.com/firewall/docs/about-security-profile-groups'
+    'Create and manage security profile groups': 'https://cloud.google.com/firewall/docs/configure-security-profile-groups'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
```

Comment:
These can be set at the top-level for the resource, as described on https://googlecloudplatform.github.io/magic-modules/develop/resource/, or inside the async block. I personally have a slight preference for top-level.

Reply:
yep, and btw I don't see the need for overriding timeouts in this case, so removing them!

Reply:
Any fields not documented in the above link (except timeouts) are unused, for example wait_ms, error, and include_project

Reply:
All clear @melinath thanks! Please, let see if it is as expected now

Reply:
@melinath I can tell `include_project: true` is definitely parsed and has effects. I elaborated more on it [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/9800#discussion_r1451424324)

Reply:
responded